dask/tests/test_sizeof.py

```diff
@@ -276,12 +277,19 @@ def test_xarray():
 
 def test_xarray_not_in_memory():
     xr = pytest.importorskip("xarray")
+    zarr = pytest.importorskip("zarr")
     np = pytest.importorskip("numpy")
     pytest.importorskip("zarr")
 
     ind = np.arange(-66, 67, 1).astype(float)
     arr = np.random.random((len(ind),))
 
+    # TODO: remove this conditional after consolidated metadata lands in v3
```

Comment:
Is this referring to landing in dask or in Zarr? Did it land? There's no issue or PR reference to check.